packages/analytics/amplify_analytics_pinpoint_dart/lib/src/analytics_plugin_impl.dart

```diff
@@ -197,6 +208,11 @@ class AmplifyAnalyticsPinpointDart extends AnalyticsPluginInterface {
     unawaited(
       Amplify.asyncConfig.then((_) {
         _endpointClient.updateEndpoint();
+        _hubEventController.add(
+          AnalyticsHubEvent.pinpointEndpointIdSet(
+            AnalyticsHubEventPayload(pinpointEndpointId: fixedEndpointId),
```

Comment:
If `updateEndpoint` fails, is it correct to fire this event?

Reply:
No as it can fail and no pinpoint id would exist, thanks fixed and added a check.